src/hdusd/engine/viewport_engine.py

```diff
@@ -348,15 +304,83 @@ def draw(self, context):
         else:
             self.notify_status("Rendering Done", "", False)
 
-    def update_render(self, scene):
-        prop = scene.hdusd.viewport
-        if self.is_gl_delegate != prop.is_gl_delegate:
-            self.is_gl_delegate = prop.is_gl_delegate
 
-            # update all scene lights since on renderer change
-            self.resync_scene_lights(scene)
+class ViewportEngineScene(ViewportEngine):
```

Comment:
What is the purpose of this class?  Can you add some documentation string?

Reply:
This is Viewport engine for rendering Blender current scene. Added comment